src/brogue/Items.c

```diff
@@ -2517,6 +2517,20 @@ void itemDetails(char *buf, item *theItem) {
                                     abs(theItem->enchant1 + 1));
                             strcat(buf, buf2);
                             break;
+                        case RING_AWARENESS:
+                            if (theItem->enchant1 > -1) {
+                                sprintf(buf2, "\n\nWith this ring equipped, you will have a %i%% chance to automatically notice adjacent traps and secret doors. (If the ring is enchanted, the chance will %s %i%%.) ",
+                                        min(100, max(0, theItem->enchant1 * 20 + 20)),
+                                        (theItem->enchant1 >= 4 ? "remain" : "increase to"),
+                                        min(100, max(0, theItem->enchant1 * 20 + 40)));
+                                sprintf(buf3, "You may occasionally notice secrets at a distance of up to %i squares away.",
+                                        (2 * theItem->enchant1) + 2);
+                                strcat (buf2, buf3);
+                            } else {
+                                sprintf(buf2, "\n\nWith this ring equipped, you can only find secret doors and traps by searching for them, and your searches are half as effective.");
```

Comment:
Is this true? You can only find by searching?

Reply:
when it's cursed, yeah

Reply:
TIL: it seems like you passive search whenever awareness is over -30, and each enchant in a ring counts for 20 awareness points. So maybe you can passive discover at -1, but with the other adjustments, it would be so rare that this seems accurate enough